### PR TITLE
Fix Orientation Issues in About and Game Activity

### LINKED ISSUE #282: Issues with Orientation Change
There are following issues with orientation change -
1. In About Screen, if any item is expanded (For ex - Why this App is needed) and orientation is changed, it becomes collapsed.
2. On changing orientation is any scenario, scenario starts from beginning again.
3. Any selection made for avatar in Store goes away on changing the orientation.

@chhavip Should I send PR for this ?

### PR DESCRIPTION
**Pull request by codingblazer**
Fixes #282 
@chhavip @aanchal07 @anubhakushwaha Changes made are as follows -

1. For About Activity - Added boolean variables isAboutGameOpen, isAboutUrgencyOpen and  isAboutHelpingOpen which will retain the state of Textview in saveInstanceState. 
Other than this, Textview needed to be made global so as to access them in onCreate().

2. For Game Activity - If the orientation is changed, activity is recreated and savedInstanceState is not null. Thus in this case, we can keep the question same as last question in SessionHistory.

Please review. Thanks :-)

### CODE REVIEW

**Comment by anubhakushwaha** [comment]
@codingblazer Try making one commit per feature,as your PR is fixing two issues please make two  commits

File: PowerUp/app/src/main/java/powerup/systers/com/AboutActivity.java [hunk 1]
```diff
@@ -7,40 +7,78 @@
...
 
+    private boolean isAboutGameOpen = false;
+    private boolean isAboutUrgencyOpen = false;
+    private boolean isAboutHelpingOpen = false;
+    private static String isGameOpen = "ABOUT_GAME_OPEN";
+    private static String isUrgencyOpen = "ABOUT_URGENCY_OPEN";
+    private static String isHelpingOpen = "ABOUT_HELPING_OPEN";
+    TextView aboutGameSection, aboutUrgencySection, aboutHelpingSection;
+
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/AboutActivity.java]: Remove extra line

File: PowerUp/app/src/main/java/powerup/systers/com/AboutActivity.java [hunk 2]
```diff
@@ -7,40 +7,78 @@
...
+        if (isAboutGameOpen){
+           aboutGameSection.setVisibility(View.VISIBLE);
+        }
+        if (isAboutUrgencyOpen){
+            aboutUrgencySection.setVisibility(View.VISIBLE);
+        }
+        if (isAboutHelpingOpen){
+            aboutHelpingSection.setVisibility(View.VISIBLE);
+        }
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/AboutActivity.java]: Remove extra line

Comment (codingblazer) [inline PowerUp/app/src/main/java/powerup/systers/com/AboutActivity.java]: Yeah Sure

File: PowerUp/app/src/main/java/powerup/systers/com/AboutActivity.java [hunk 3]
```diff
@@ -7,40 +7,78 @@
 
 public class AboutActivity extends Activity {
 
+    private boolean isAboutGameOpen = false;
+    private boolean isAboutUrgencyOpen = false;
+    private boolean isAboutHelpingOpen = false;
+    private static String isGameOpen = "ABOUT_GAME_OPEN";
+    private static String isUrgencyOpen = "ABOUT_URGENCY_OPEN";
+    private static String isHelpingOpen = "ABOUT_HELPING_OPEN";
+    TextView aboutGameSection, aboutUrgencySection, aboutHelpingSection;
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/AboutActivity.java]: Declare private if not needed outside this class.

Comment (codingblazer) [inline PowerUp/app/src/main/java/powerup/systers/com/AboutActivity.java]: Yeah

**Comment by codingblazer** [comment]
@chhavip Done. Please Review.

**Comment by chhavip** [comment]
Merging. Looks good. Thanks @codingblazer

**Pull request merged**